src/components/modules/MainPage/LongCard/StyledLongCard.js

```diff
@@ -0,0 +1,66 @@
+import styled, { css } from "styled-components";
+
+export const Container = styled.div`
+  width:100rem;
+  height: 17.5rem;
+  height:100%;
```

Comment:
this doesn't affect anything, always avoid to manually add height.

Reply:
<img width="788" alt="Screen Shot 2022-04-11 at 2 40 35 PM" src="https://user-images.githubusercontent.com/58893691/162838218-6acf9adb-90d4-4c95-a482-ff4282f01644.png">
266.58 px ---> 16.00rem

Reply:
I put 17.5 rem Height for the Container

Reply:
anyway deleted already !